src/avs/AVSList.jsx

```diff
@@ -171,122 +172,131 @@ export default function AVSList() {
           }
         />
       </div>
-      <div className="flex flex-1 flex-col rounded-lg border border-outline bg-content1 text-sm">
-        <Table
-          aria-label="AVS list"
-          layout="fixed"
-          removeWrapper
-          classNames={{
-            base: 'overflow-x-auto h-full',
-            table: 'h-full'
-          }}
-          sortDescriptor={state.sortDescriptor}
-          onSortChange={e => dispatch({ sortDescriptor: e })}
-        >
-          <TableHeader columns={columns}>
-            {column => (
-              <TableColumn
-                allowsSorting
-                className={`bg-transparent py-4 text-sm font-normal leading-5 text-foreground-active data-[hover=true]:text-foreground-2 ${column.className}`}
-                key={column.key}
-              >
-                {column.label}
-              </TableColumn>
-            )}
-          </TableHeader>
-          <TableBody
-            emptyContent={
-              <div className="flex flex-col items-center justify-center">
-                <span className="text-lg text-foreground-2">
-                  No result found for &quot;
-                  {state.searchTerm?.length > 42
-                    ? `${state.searchTerm?.substring(0, 42)}...`
-                    : state.searchTerm}
-                  &quot;
-                </span>
-              </div>
-            }
+
+      {!state.isFetchingAvsData && state.error && (
+        <div className="flex flex-1 flex-col items-center justify-center rounded-lg border border-outline bg-content1 text-sm">
+          {state.error.message === 'No AVS found' ? (
+            <div className="flex flex-col items-center justify-center">
+              <span className="text-lg text-foreground-2">
+                No result found for &quot;
+                {state.searchTerm?.length > 42
+                  ? `${state.searchTerm?.substring(0, 42)}...`
+                  : state.searchTerm}
+                &quot;
+              </span>
+            </div>
+          ) : (
+            <ErrorMessage error={state.error} />
+          )}
+        </div>
+      )}
+
+      {!state.error && (
+        <div className="flex flex-1 flex-col rounded-lg border border-outline bg-content1 text-sm">
+          <Table
+            aria-label="AVS list"
+            layout="fixed"
+            removeWrapper
+            classNames={{
+              base: 'overflow-x-auto h-full',
+              table: 'h-full'
```

Comment:
Not related to this PR, but I noticed a weird thing and couldn't find an "official" way to get rid of it. Changed the colors, so you can spot it:

![image](https://github.com/user-attachments/assets/12302887-8ea4-4536-b910-9ac7675c8d8c)

This empty space is on each table. @vincenthongzy, you need to apply this to your tables as well.

```suggestion
              table: 'h-full',
              thead: '[&>tr:last-child]:hidden'
```

Reply:
Added suggested change to the table component.